spec/lib/msf/core/modules/loader/base_spec.rb

```diff
@@ -357,7 +357,7 @@ class MetasploitModule < Msf::Auxiliary
           allow(module_manager).to receive(:on_module_load)
 
           # if the module eval error includes the module_path then the module_path was passed along correctly
-          expect(subject).to receive(:elog).with(/#{Regexp.escape(module_path)}/)
+          expect(subject).to receive(:elog).with(/#{Regexp.escape(module_path)}/, error: an_instance_of(NoMethodError))
```

Comment:
The error is from the malformed module content intentionally having a mistake:

```
        # purposeful typo to check that module path is used in backtrace
        inclde Exploit::Remote::Tcp
```

Which results in a NoMethodError being raised